app-server/database.js

```diff
@@ -0,0 +1,771 @@
+"use strict";
+
+import mysql from "mysql";
+import config from "../config/config.json";
+
+function getMysqlConnection() {
+  var connection = mysql.createConnection({
+    host: config.db.host,
+    user: config.db.user,
+    password: config.db.password,
+    database: config.db.databaseName
+  });
+  connection.connect();
+  return connection;
+}
```

Comment:
You might want to take connections from a shared connection pool, rather than creating a new one for each query. Did a brief search and found [`node-pool`](https://github.com/coopernurse/node-pool#step-1---create-pool-using-a-factory-object), but no idea what the most used libraries are for that sort of thing.